packages/react-sdk/MIGRATING.md

```diff
@@ -0,0 +1,137 @@
+# Migrating
```

Comment:
I was thinking may be call the file `MIGRATION.md`. That sounded more correct. Are there examples of calling it `MIGRATING.md` or `MIGRATION.md` out there?

No big issues. Was just curious. I used the analogy of `CONTRIBUTING.md` to name this `MIGRATING.md`.

Reply:
Sounds good. I don't have any examples of MIGRATION.md or MIGRATING.md.